ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomCommandingDemoController.swift

```diff
@@ -56,11 +56,18 @@ class BottomCommandingDemoController: UIViewController {
         CommandingItem(title: "Boolean Item " + String($0), image: homeImage, action: commandAction, isToggleable: true)
     }
 
+    private lazy var badgeCommand: CommandingItem = {
+        let badge = BadgeView(dataSource: BadgeViewDataSource(text: "Badge"))
+        let stackView = UIStackView(frame: CGRect(x: 0, y: 0, width: 60, height: 25))
```

Comment:
this seems wierd. even the tableviewcell demo cases, we pass in random things like label right? is there a different bug in cell rendering? or is the badgeView intrinsic content size isn't set correctly taht the calculation of the cell not working right? @sophialee0416

Reply:
While I was testing, there was definitely something weird going on with both TableViewCell and BadgeView, I'll followup with Sophia.

Reply:
Offline we agreed to merge these changes and file a separate bug for TableViewCell.